deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts

```diff
@@ -255,6 +256,14 @@ export class MigrationConsoleStack extends MigrationServiceCore {
             ]
         })
 
+        const getSecretsPolicy = new PolicyStatement({
+            effect: Effect.ALLOW,
+            resources: [props.servicesYaml.target_cluster.basic_auth?.password_from_secret_arn ?? ""],
+            actions: [
+                "secretsmanager:GetSecretValue"
+            ]
+        });
```

Comment:
Can we please either wrap this in a conditional or use a cdk.CfnConditional so that this isn't generated in the event there is no ARN?  I can see a user (or security reviewer) being surprised to see this in the Cfn stack def if they haven't done anything w/ Secrets.